classes/PodsI18n.php

```diff
@@ -196,4 +205,240 @@ private static function default_strings() {
 
 	}
 
+	/**
+	 * Get current language information from Multilingual plugins
+	 *
+	 * @since 2.6.6
+	 * @since 2.7 Moved to this class from PodsAPI
+	 *
+	 * @param array $args (optional) {
+	 *     @type bool $redo Rerun logic?
+	 * }
+	 *
+	 * @return array
+	 */
+	public static function get_current_language( $args = array() ) {
```

Comment:
Maybe leave `get_current_language()` as it is and add `get_current_locale()`.
And reflect this to the properties as well:
`$current_language` => `$current_locale`
`$current_language_data` => `$current_language`